packages/e2e/alonzo-private-network/scripts/download-config.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script is for development purpose only. It will download official config files so we can compare them with current config files.
```

Comment:
```suggestion
# This script is for development purposes only. It will download IOG-maintained config files so we can compare them with current config files.
```

Reply:
fixed here https://github.com/input-output-hk/cardano-js-sdk/commit/98b89bd44777a9dd58080a01a52d04221779d467